src/test/groovy/nu/studer/gradle/buildscan/teamcity/TeamCityBuildScanPluginTest.groovy

```diff
@@ -115,8 +117,20 @@ class TeamCityBuildScanPluginTest extends Specification {
         !outputContainsExpectedMessage(result.output)
     }
 
-    private void configurePluginClasspath(String classpathSystemPropertyName) {
-        runner.withPluginClasspath(PluginUnderTestMetadataReading.readImplementationClasspath() + [new File(System.getProperty(classpathSystemPropertyName))])
+    def "no service messages emitted when build scan or Gradle Enterprise plugin are not on the classpath"() {
+        given:
+        configurePluginClasspath()
+        applyPlugin()
```

Comment:
@facewindu Does this test implementation really match the description of the test? If so, why ar we configuring the plugin classpath and applying the plugin?

Reply:
I reworked it to be clearer. We were adding the TC plugin to the classpath, but not the GE/build scan plugin (as no parameters were given to configurePluginClasspath() )